src/PowerShellEditorServices/Services/PowerShellContext/PowerShellContextService.cs

```diff
@@ -32,10 +32,18 @@ namespace Microsoft.PowerShell.EditorServices.Services
     /// </summary>
     internal class PowerShellContextService : IHostSupportsInteractiveSession
     {
-        private static readonly string s_commandsModulePath = Path.GetFullPath(
+        private static string s_bundledModulesPath = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "..", "..");
+
+        private static string s_commandsModulePath => Path.GetFullPath(
+            Path.Combine(
+                s_bundledModulesPath,
+                "PowerShellEditorServices",
+                "Commands",
+                "PowerShellEditorServices.Commands.psd1"));
+        private static string _psReadLineModulePath => Path.GetFullPath(
             Path.Combine(
-                Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location),
-                "../../Commands/PowerShellEditorServices.Commands.psd1"));
+                s_bundledModulesPath,
```

Comment:
This is the meat of the change. We are using a path relative the executing assembly to find the module path instead of using the one provided by the user.